downstream/modules/platform/proc-configure-pac-settings.adoc

```diff
@@ -0,0 +1,29 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-08
+:_mod-docs-content-type: PROCEDURE
+
+[id="configure-pac-settings_{context}"]
+= Configuring policy enforcement settings
+
+You can specify how your {PlatformNameShort} instance interacts with OPA by modifying your global settings.
+
+.Procedure
+. From the navigation panel, select {MenuSetPolicy}.
+. Click *Edit policy settings*.
+. On the Policy Settings page, fill out the following fields: 
+.. *OPA Server hostname*: Enter the name of the host that connects to the OPA service.
```

Comment:
This could be a definition list too. You'll want to include + before and after to maintain numbering.